salt/states/file.py

```diff
@@ -4570,6 +4570,372 @@ def replace(name,
     return ret
 
 
+def keyvalue_list(
```

Comment:
The name of this function feels a little strange to me. Would it be more straightforward if it were just `file.keyvalue`?

Reply:
It is, and that's exactly what I'm doing in the other function below called exactly that. (https://github.com/saltstack/salt/pull/50627/files#diff-3b447dc835943beb1326f98f684fe783R4905-R4936)

 I'd written the first later, then figured it'd be nice to give a list. Since I didn't want to call the original function over and over I'd figured the shorthand version could instead call the list version.

I'd be happy calling it ``file.keyvalues``, and do away with the shorthand version though.

Reply:
That makes a lot of sense. Excellent idea!